fiftyone/operators/decorators.py

```diff
@@ -85,6 +87,8 @@ def wrapper(*args, **kwargs):
 def dir_state(dirpath):
     if not os.path.isdir(dirpath):
         return None
-    return max(
-        os.path.getmtime(os.path.join(dirpath, f)) for f in os.listdir(dirpath)
-    )
+    # use glob instead of os.listdir to ignore hidden files (eg .DS_STORE)
+    dirs = glob.glob(os.path.join(dirpath, "*"))
```

Comment:
I defer to @ritch's judgement on this hidden file change. This isn't necessary to fix the bug, and while it would be strange to use hidden files for important things, it *could* be the case that a plugin has a hidden file whose update *should* trigger a plugin reload.

Reply:
the only place it is used is for the plugin_cache and finding the latest updated timestamp in the plugins dir. Any hidden files updated by a plugin should be inside the plugin's directory but not the fo.config.plugins_dir right?

Reply:
Oh right I forgot we aren't recursively listing. Just hoping that the directory's `mtime` is sufficient. Either way, defer to Ritchie's review